mlflow/store/file_store.py

```diff
@@ -46,6 +45,11 @@ def _make_persisted_run_info_dict(run_info):
     # old mlflow versions to read
     run_info_dict = dict(run_info)
     run_info_dict['tags'] = []
+    run_info_dict['name'] = ''
```

Comment:
Makes sense, I think this change is necessary e.g. for MLflow projects running older MLflow versions to be able to properly deserialize the run. We should QA this pre-1.0 though :P